EcoSoapBank/Helpers/Randomizable.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Randomizable.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-26.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+protocol Randomizable {
+    static func random() -> Self
+}
+
+extension Randomizable where Self: CaseIterable, AllCases == [Self] {
```

Comment:
Very nice. I always like seeing your little utility helpers! You come up with some great ones.